docs/src/development/ssh.md

```diff
@@ -2,131 +2,118 @@
+You can use key pairs to access deployed environments if you are setting up an automation tool or prefer that method.
 
-To connect using certificate-based authentication, install the [Platform.sh CLI](/development/cli/_index.md).
+Automation tools may also use [API tokens](/development/cli/api-tokens.md).
 
-Once installed, you may run `platform login` or any CLI command that would require authentication.  In either case, a browser window will open and ask you to login with your Platform.sh account credentials.  This web page is already encrypted with TLS over HTTP, making it secure.
+## Authenticate with certificates
 
-The login process will issue a certificate that gets stored in your local SSH configuration.  The certificate is automatically cycled every hour for a new certificate as long as your session is active.  If you are inactive for an extended period your certificate will expire, and the system will ask you to login again the next time you use a command that requires authentication.
+To connect using certificates:
 
-## Keypair-based authentication
+1. Install the [Platform.sh CLI](/development/cli/_index.md).
+2. Run `platform login`.
+4. In the open browser window, log in with your Platform.sh account credentials. (This webpage is encrypted with HTTPS [HTTP over TLS], making it secure.)
+5. Authorize the CLI to use your account.
 
-This process requires two [RSA keys](https://en.wikipedia.org/wiki/RSA_%28cryptosystem%29):
+A certificate gets stored in your local SSH configuration. The certificate is automatically cycled every hour for a new certificate as long as your session is active.
 
-* A **private key** kept secret by the user
-* A **public key** stored within the Platform.sh account
+If you are inactive for an extended period your certificate expires and you are asked to login again the next time you use a command that requires authentication.
 
-These keys are called the *public-private keypair* and usually look like random lines of characters, like this:
+## Authenticate with a key pair
 
-*A private key*:
+This process requires two keys:
+
+* A **private key** you must keep _secret_
+* A **public key** stored in your Platform.sh account
+
+These keys usually look like random lines of characters, like this example of [RSA keys](https://en.wikipedia.org/wiki/RSA_%28cryptosystem%29):
+
+A private key:
 
 ```text
 -----BEGIN RSA PRIVATE KEY-----
 MIIEowIBAAKCAQEAtpw0S4DwDVj2q04mhiIMkhvrYU7Z6hRiNbTFsqg3X7x/uYS/
 dcNrSvT82j/jSeYQP3Dsod9GERW+dmOuLaFNeiqOStZi6jRSWo41hCOWOFbpBum3
-ra1n6nUO1wa/7O5wbgzhUOfnim77oOK0UgkqPArBCNXiNFTUJAvRyVmCtvJOyrqz
-...(20 more lines like this)...
-cPjJ/wKBgGd3eZIBK6Ak92u65HYXgY9EcX3vBNP4NsF087uxV4YfrM18KlGf5I87
+...(many more lines like this)...
 QGerp3VKaGe0St3ot57GlwCAQUJAf1mit8qDTi0I8MhBe7q2lstXkBvde7GY1gKx
 Kng4ohG6xHZ/OvC9tq7/THwAvleaxgLZN5GyXfAqNylDdZ0LtSjl
 -----END RSA PRIVATE KEY-----
 ```
 
-*A public key (one very long line)*:
+A public key (one very long line):
```

Comment:
I love that we keep the example on the public key, I'd describe it a bit differently, as long as it doesn't require changing the link here:

```suggestion
A public key (ends with your email):
```

Should this content be here or in the section below to identify their public `.pub` key?

Reply:
I would probably move it down after removing the private key. I would note if it's there, it could be about finding existing keys and there might not necessarily end with an email address.